pkg/operator/controller/controller.go

```diff
@@ -64,6 +65,15 @@ func New(mgr manager.Manager, config Config) (controller.Controller, error) {
 	if err := c.Watch(&source.Kind{Type: &operatorv1.DNS{}}, &handler.EnqueueRequestForObject{}); err != nil {
 		return nil, err
 	}
+	if err := c.Watch(&source.Kind{Type: &appsv1.DaemonSet{}}, &handler.EnqueueRequestForOwner{}); err != nil {
+		return nil, err
+	}
+	if err := c.Watch(&source.Kind{Type: &appsv1.DaemonSet{}}, &handler.EnqueueRequestForOwner{}); err != nil {
```

Comment:
You establish two watches on `DaemonSet`.

Reply:
Fixed and squashed